src/main/java/com/iota/iri/service/transactionpruning/SizePruningCondition.java

```diff
@@ -88,7 +88,7 @@ public int getSnapshotPruningMilestone() throws TransactionPruningException {
         int initialIndex;
         try {
             Pair<Indexable, Persistable> ms = tangle.getFirst(Milestone.class, IntegerIndex.class);
-            if (ms == null) {
+            if (ms == null || ms.low == null) {
```

Comment:
A question, the current annoying code actually returns null?
This is surprising I thought it just returned an empty object...

Reply:
no it doesnt

Reply:
to clarify, ms is never null, ms.low can be null